subh.ftg

```diff
@@ -0,0 +1,19 @@
+Countdown time in Python
+
+
+
+
+
+import time
+
+def countdown(time_sec):
+    while time_sec:
+        mins, secs = divmod(time_sec, 60)
+        timeformat = '{:02d}:{:02d}'.format(mins, secs)
```

Comment:
.format was popular in python 2.x, if you want you can do the same with `f string`.
```py
timeformat = f'{mins:02d}:{secs:02d}'